public/app/features/teams/TeamGroupSync.tsx

```diff
@@ -11,8 +11,9 @@ import { getTeamGroups } from './state/selectors';
 import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
 import { CloseButton } from 'app/core/components/CloseButton/CloseButton';
 
-function mapStateToProps(state: StoreState) {
+function mapStateToProps(state: StoreState, props: OwnProps) {
   return {
+    isReadOnly: props.isReadOnly,
```

Comment:
You don't really need to do this if you don't make any changes to props/ use them to modify state, the props will be just passed to the component without needing to go through `mapStateToProps`.

Reply:
Got it thanks! Made the suggested change :tada: